akka-projection-r2dbc/src/main/scala/akka/projection/r2dbc/internal/R2dbcProjectionImpl.scala

```diff
@@ -1101,7 +1149,7 @@ private[projection] class R2dbcProjectionImpl[Offset, Envelope](
         offsetStore
           .saveOffsets(offsets)
           .map { done =>
-            val batchSize = acceptedContexts.map { _.groupSize }.sum
+            val batchSize = batch.map { _.groupSize }.sum
```

Comment:
When testing, noticed that the Cinnamon metric for number of envelopes that have had offsets stored was much lower than expected. The `acceptedContexts` that was used before is the envelopes filtered by those that are in-flight, but if there are multiple events from one persistence id in the batch, only the last one is in-flight. Fixed by using the size of the actual batch.